introduction.md

```diff
@@ -1,2 +1,9 @@
 This is the introcution to dojo_rules project.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?